arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,6 +410,11 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current state of Analysis Services resource. It allows more custom terminal state like Paused."
```

Comment:
"...It allows more custom terminal state like Paused." Terminal states are already covered by ProvisioningState. so change the description to be something like "the state variable is used to indicate the current state of the given Analysis Services server."